.travis.yml

```diff
@@ -21,25 +18,13 @@ env:
 before_install:
   - sudo apt-get update -yq
   - sudo sh testing/getmsfonts.sh
-  - if [[ "$PYTHON" == "2.7" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
-    else
-      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-    fi  
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
   - hash -r
   - conda config --set always_yes yes --set changeps1 no 
   - conda config --add channels conda-forge
-  # Pinned deps aren't happy when conda>4.6.14
-  - if [[ "$DEPS" == "pinned" ]]; then
-      conda install -q conda=4.6.14;
-    else
-      conda update -q conda;
```

Comment:
this line is required to update conda to the latest version. it used to fail on pinned and py27 but should be ok with py27 removal